src/media/js/main.js

```diff
@@ -151,7 +151,9 @@ function(_) {
         // are in a packaged app, wait for the iframe to be loaded, otherwise
         // we are using the direct installer and we just need to wait for the
         // normal loaded event.
-        var event_for_apps = window.location.protocol === 'app:' ? 'iframe-install-loaded' : 'loaded';
+        var is_packaged_app = window.location.protocol === 'app:';
+        var is_iframed_app = window.top != window.self;  // Crude, but enough for our needs.
```

Comment:
I updated to use `!==`. I've filed another bug to clean up the iframe checks, it will be easier as we might need to cherry-pick this one. https://bugzilla.mozilla.org/show_bug.cgi?id=1117771